oak/server/oak_node.h

```diff
@@ -40,20 +39,20 @@ class OakNode final : public ::oak::Node::Service {
   void InitEnvironment(wabt::interp::Environment *env);
 
   // Native implementation of the `oak.read_method_name` host function.
-  ::wabt::interp::HostFunc::Callback OakReadMethodName(wabt::interp::Environment *env);
+  ::wabt::interp::HostFunc::Callback OakReadMethodName(wabt::interp::Environment* env);
```

Comment:
Uhmm, how come this was not formatted by `scripts/format` in my PR?

Reply:
Failure to rebase correctly.